dist/openQA_mail_notification.rb

```diff
@@ -1,37 +1,42 @@
+#!/usr/bin/ruby
+
 require 'net/https'
 require 'net/smtp'
 require 'uri'
 require 'json'
+require 'mail'
 
 FROM = 'obs-admin@opensuse.org'
-TO = 'obs-test@opensuse.org'
+TO = 'obs-tests@opensuse.org'
 SMTP_SERVER = ''
-OPEN_QA = 'https://openqa.opensuse.org/api/v1'
+OPEN_QA = 'https://openqa.opensuse.org/'
+OPEN_QA_API = "#{OPEN_QA}api/v1"
```

Comment:
Not a big deal, but since this is just used once you could also put the 'api/v1' suffix into line 20 and remove this constant.